comments/pylint/refactor.md

```diff
@@ -0,0 +1,12 @@
+# pylint refactor
+
+**Line %{lineno}** [ _%{code}_ ]  :  %{message}.
+ Was reported.
+
+This code is emitting a [`code smell`][code smell], and may be in need of
+a re-write or refactor.
```

Comment:
```suggestion
This code is emitting a [`code smell`][code smell], and may be in need of
a rewrite or refactor.
```

Correction as suggested from The Collaborative International Dictionary of English, but WordNet acknowledges that `re-write` is also correct.  Not sure it needs to be corrected.  But.. is this file one of the ones are are doing "one sentence per line"?

Reply:
>  is this file one of the ones are are doing "one sentence per line"?
yes.  So thank you for catching that.  Although, this one matters a bit less than the website-copy versions, since these are used internally for testing/dev.

Reply:
Still benefit from "less diff noise" when changes happen though.